test/lit.cfg

```diff
@@ -531,7 +531,7 @@ shutil.rmtree(completion_cache_path, ignore_errors=True)
 ccp_opt = "-completion-cache-path %r" % completion_cache_path
 lit_config.note("Using code completion cache: " + completion_cache_path)
 
-config.swift_host_lib_dir = make_path(config.swift_lib_dir, 'swift', 'host')
+config.swift_host_lib_dir = config.swift_bin_dir
```

Comment:
Could you add `%swift-host-plugin-dir`?
For Darwin/Linux/etc. it'd be `make_path(config.swift_lib_dir, 'swift', 'host', 'plugins')`
For Windows, `config.swift_bin_dir`

Reply:
Yeah, except I think that we should extract that change.